includes/functions.php

```diff
@@ -557,15 +557,16 @@ function isSNMPable($device)
 {
     global $config;
 
-    $pos = snmp_get($device, "sysObjectID.0", "-Oqv", "SNMPv2-MIB");
-    if (empty($pos)) {
-        // Support for Hikvision
-        $pos = snmp_get($device, "SNMPv2-SMI::enterprises.39165.1.1.0", "-Oqv", "SNMPv2-MIB");
-    }
-    if ($pos === '' || $pos === false) {
-        return false;
-    } else {
+    $pos = snmp_check($device);
+    if ($pos === true) {
         return true;
+    } else {
```

Comment:
For now it's catch all. Whilst I'm sure this change will work I can't be 100% sure, all I've been able to test against is net-snmpd and snmpsimd. Up / down is the bit I know works but actually checking we have sysObjectId is where I've had limited testing.

We can remove it but I just didn't want to.